### ISSUE TITLE
Fetching the wrong screenshot.

### ISSUE THREAD

**Issue by chriscm2006** [dequelabs/Deque-University-for-Android#35]
The issue is that as a part of the automated journey to uncover accessibility violations; we call the attest api for each pages in the journey. After the api call we are accessing /api/results to get the json response to create html report associated with the page. At the same time we are accessing /api/screenshot.html to map to the html report created from the json. So for the journey of four pages we get the same screenshot which is for the first page of the journey. I hope you understood the issue; happy to call and explain.

**Comment by chriscm2006** [comment]
This sounds like an ordering issue. 

`/api/screenshot.html`

Will return the last location that was analyzed.

`/api/results`

Will perform an analysis.

If you want the screenshot of the analyzed content you need to perform an analysis and then fetch the screenshot.

**Comment by chriscm2006** [comment]
@nmashni Let me know if this resolves the clients issues in this case and I will close.

**Comment by chriscm2006** [comment]
Closing as incomplete. Will reopen if we get more information.
